supervisor/qemu/config.example.toml

```diff
@@ -0,0 +1,47 @@
+[base]
+supervisor_id = "7d55ec6d-15e7-4b84-8c04-7c085fe60df4"
+coord_connector = "cli_connector"
+
+[cli_connector]
+# Doesn't do anything yet, but we require the images key to be present
+images = { }
+
+[image_store]
+# Doesn't do anything yet. This endpoint will be used to request
+# images to be downloaded etc. later on:
+http_endpoint = "https://localhost:8080"
+
+# Local mountpoint of the read-only image store:
+fs_endpoint = "./image-store"
+
+[qemu]
+qemu_binary = "qemu-kvm"
+qemu_img_binary = "qemu-img"
+
+state_dir = "./state"
+
+qemu_args = [
```

Comment:
Perhaps we can later include an example configuration for UEFI that has the OVMF vars file path and extra qemu args